### PR TITLE
bedtools, oc, sensible-editor, azcopy: fix typos and add oc project example

### PR DESCRIPTION
**Pull request by siavashsoleymani**
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/oc.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 `oc new-project {{project_name}}`
 
+- Change project to project_name:
```
Comment (navarroaxel) [inline pages/common/oc.md:15]: Based on `git-checkout.md` and `git-switch.md` pages:
```suggestion
- Switch to an existing project:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @siavashsoleymani, just 1 comment.

**Comment by siavashsoleymani** [comment]
@navarroaxel agree with you, thanks for point out. Done!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Great catches - thanks, @siavashsoleymani!

**Comment by siavashsoleymani** [comment]
@sbrl it's my pleasure your welcome

**Pull request merged**